packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -201,6 +201,15 @@ export class DetailsListBase extends BaseComponent<IDetailsListProps, IDetailsLi
     if (this.props.onDidUpdate) {
       this.props.onDidUpdate(this);
     }
+
+    if (prevProps.dragDropEvents !== this.props.dragDropEvents) {
+      this._dragDropHelper = this.props.dragDropEvents
+        ? new DragDropHelper({
+            selection: this.props.selection || new Selection({ onSelectionChanged: undefined, getKey: this.props.getKey }),
```

Comment:
If we were to pursue this change, should we keep the member variable `this._selection` in sync?

https://github.com/OfficeDev/office-ui-fabric-react/blob/aab8f62851d91184b3c2ac7758dae2bc822e40bb/packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.base.tsx#L116

Reply:
Perhaps we can consolidate the creation of `DragDropHelper` as part of this change.

Reply:
@KevinTCoughlin Can @aditima advice? 

BTW instead of `null`, it would be good if both the original code in the constructor and over here uses `undefined`, which I think will eliminate the need for the hacky `!` on line 538.

We should also dispose of the original _dragDropHelper before we reassign, or there will otherwise be a memory leak.